scripts/us_fed/treasury_constant_maturity_rates/extract_data.py

```diff
@@ -0,0 +1,46 @@
+'''
+Extracts the data portion out of the constant maturity rate csv file downloaded
+from Federal Reserve.
+'''
+
+USAGE = '''
+python3 extract_data.py [path]
+    "path": Path to the raw csv containing rates at all maturities.
+
+Example: python3 extract_data.py FRB_H15.csv
+    This will extract the data out of "FRB_H15.csv" and store it in
+    "treasury_constant_maturity_rates.csv".
+    The output table has the same number of columns as the number of constant
+    maturities provided and an extra column for dates.
+    "date" column is of the form "YYYY-MM-DD".
+    The other interest rate columns are numeric.
+'''
+
+
+import sys
+
+from maturity import MATURITIES
+import pandas as pd
+
+
+def main():
+    if len(sys.argv) != 2:
+        print(USAGE, file=sys.stderr)
+        return
+
+    out_df = pd.DataFrame()
+    header_rows = 5
```

Comment:
Can use pd read_csv's skiprows parameter instead

Reply:
skiprows also skips the header https://stackoverflow.com/questions/27325652/python-pandas-read-csv-skip-rows-but-keep-header